### PR TITLE
[PLAYER-3859] Muted state now overrides volume for VoiceOver (Narrato…

### PR DESCRIPTION
**Pull request by zelibobla**
Fixed volume control attribute to reflect muted state (if muted) over the volume value.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ node_modules
 build
 coverage
 .idea
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode/
```
Comment (jpfranco) [inline .gitignore]: Is updating the skin config submodule (below) deliberate? I don't see any changes on the diffs for that

File: js/components/volumeControls.js [hunk 2]
```diff
@@ -92,6 +92,9 @@ var VolumeControls = createReactClass({
    * @returns {String} The current volume in a screen reader friendly format (i.e. 20% volume).
    */
   getAriaValueText: function() {
+    if (this.props.controller.state.volumeState.muted) {
```
Comment (jpfranco) [inline js/components/volumeControls.js]: Can we add a unit test that checks that the correct label is returned?

File: package.json [hunk 3]
```diff
@@ -23,55 +23,55 @@
...
+    "classnames": "^2.2.6",
     "create-react-class": "^15.6.3",
     "deepmerge": "^1.4.0",
     "lodash.debounce": "4.0.3",
     "lodash.values": "4.1.0",
-    "prop-types": "^15.6.1",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
+    "prop-types": "^15.6.2",
+    "react": "^16.5.2",
```
Comment (jpfranco) [inline package.json]: Good idea to take the opportunity to update to latest React, thanks

File: package.json [hunk 4]
```diff
@@ -23,55 +23,55 @@
...
     "react-scrollbar": "^0.5.4",
     "screenfull": "3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.51",
-    "@babel/preset-env": "^7.0.0-beta.51",
-    "@babel/preset-react": "^7.0.0-beta.51",
+    "@babel/core": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
```
Comment (aeng7) [inline package.json]: Should we update this to 7.1.0 as well? To keep in sync with the other babel presets.

File: .gitignore (hunk 1 above)
Comment (zelibobla) [reply .gitignore]: It's just automatically updated due to latest master version is newer than the one is referred from .gitmodules. But particular files being pulled via .gitmodules settings are not changed, so this update actually makes no sense. I can roll it back if necessary.

File: js/components/volumeControls.js (hunk 2 above)
Comment (zelibobla) [reply js/components/volumeControls.js]: done

File: package.json (hunk 4 above)
Comment (zelibobla) [reply package.json]: done

File: .gitignore (hunk 1 above)
Comment (jpfranco) [reply .gitignore]: Oh ok no problem. If it's the latest from master then it's fine

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**